app/models/custom_attributes/dropdown_value.rb

```diff
@@ -0,0 +1,25 @@
+module CustomAttributes
+  module DropdownValue
+    def value=(custom_attribute_option_id)
+      custom_attribute_option = custom_attribute_options.find_by id: custom_attribute_option_id
+      if (custom_attribute_option_value = custom_attribute_option_values.first).present?
+        if custom_attribute_option.present?
+          custom_attribute_option_value.custom_attribute_option = custom_attribute_option
+          custom_attribute_option_value.save
+        else
+          custom_attribute_option_value.destroy
```

Comment:
redmine for mandatory check and add it in December

Reply:
http://pm.7vals.com/issues/9485